AzureCalling/External/Calling/AudioDeviceDataModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+struct AudioDeviceDataModel {
+    let image: UIImage
+    let name: String
+    let enabled: Bool
+}
```

Comment:
This is not really a data model since it has a UIImage. I think we can call it `AudioDeviceOption` and just define it as part of the `AudioDeviceSelectionViewContoller`.